core/store/migrate/migrations/0130_pipeline_runs_brin_to_btree.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+	-- Migrate idx_pipeline_runs_created_at to BTREE
+    DROP INDEX IF EXISTS idx_pipeline_runs_created_at;
+    CREATE INDEX idx_pipeline_runs_created_at ON public.pipeline_runs USING BTREE (created_at);
+
+	-- Migrate idx_pipeline_runs_finished_at to BTREE
+    DROP INDEX IF EXISTS idx_pipeline_runs_finished_at;
+    CREATE INDEX idx_pipeline_runs_finished_at ON public.pipeline_runs USING BTREE (finished_at);
+
+    -- Migrate idx_pipeline_runs_pipeline_spec_id to HASH index
+    DROP INDEX IF EXISTS idx_pipeline_runs_pipeline_spec_id;
+    CREATE INDEX idx_pipeline_runs_pipeline_spec_id ON public.pipeline_runs USING HASH (pipeline_spec_id);
```

Comment:
Yes, in fact the BTREE index was counterproductive in that the query ran noticeably faster without an index than with it.  This makes sense theoretically as well, since BTREE's work best when all the keys are unique, and perform worse and worse as the ratio between total # of rows to unique rows becomes large.  So whenever the number of pipelines is much larger than the number of jobs defined the BTREE will only add overhead.  HASH is designed for exactly this use case.